### ISSUE TITLE
[IOS] HTTP load failed (error code: -999 [1:89]) on UpdateMap or Zoom or any other stuff with tiles

### ISSUE THREAD

**Issue by timskap** [mapbox/mapbox-unity-sdk#1302]
* Unity version: 2018.3.0f1
  * Scripting Runtime Version: .Net 4.x
  * Scripting Backend: IL2CPP
  * Api Compatibility Level: .net 4.x
* Mapbox SDK version: 2.0.0
* The platform you're building to: iOS
* A description of what you're trying to do:

I've got a problem with Update Map and getting new tiles. 

Firstful I've got that error: 
`no 'ETag' header present in response for `

But then when I tried to dig deeper and add additional variables for debugging I found another error: 
`HTTP load failed (error code: -999 [1:89])`

As you can see; -999 is caused by ErrorCancelled. This means: another request is made before the previous request is completed. 

I think that problem exist because one request for new tile didn't canceled after new request with another tile. At Android I don't see that problem but on iOS it's get me a headache already for one week. If I zoom or move camera slowly, error is not showing.

* Steps to recreate the bug if appropriate: UpdateMap with new Zoom or move Camera with Boundaries settings.

**Comment by TimNedvyga** [comment]
Same :(

**Comment by juliengdt** [comment]
This is not due to MapBox, but iOS "new" system-wide logging, including CFNetwork framework.
This log appears when a request is cancelled. (Mapbox may cancel some for Data efficiency I imagine)

TL;DR: don't worry

**Comment by ask-a-lie** [comment]
Hello, 

> `no 'ETag' header present in response for`

I think this error is due to difference between upper and lower case, that is, ETag and Etag.
 
https://github.com/mapbox/mapbox-unity-sdk/issues/913#issuecomment-485269666

**Comment by wojciech-kulik** [comment]
Finally, I've found a way to disable those annoying logs:
https://stackoverflow.com/questions/37800790/hide-strange-unwanted-xcode-logs

Just set environment variable `OS_ACTIVITY_MODE=disable` in schema options.
